modules/disable-node-caching-in-elyra.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: PROCEDURE
+
+[id="disable-node-caching-in-elyra_{context}"]
+= Disabling node caching in Elyra
+
+[role='_abstract']
+Elyra automatically caches the ouputs of your components, or "nodes". You can also disable this feature if you want to ensure a fresh execution for each component instead of resuing outputs from previous runs.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+
+* You have created a data science project.
+* You have created and configured a pipeline server within the data science project that contains your workbench.
++
+[IMPORTANT]
+====
+To ensure that the runtime configuration is created automatically, you must create the pipeline server before you create the workbench.
```

Comment:
add "a default", remove "auromatically"
 To ensure that **a default** runtime configuration is created,